util/chplenv/compiler_utils.py

```diff
@@ -14,7 +14,9 @@
 @memoize
 def get_compiler_version(compiler):
     version_string = '0'
-    if 'gnu' in compiler:
+    if compiler == 'aarch64-gnu':
+        version_string = run_command(['aarch64-unknown-linux-gnu-gcc', '-dumpversion'])
```

Comment:
I should note that only PrgEnv-cray and the unwrapped cross gcc are available for ARM, so we can't use the `cc` wrapper.  Progeny-gnu does not exist, and PrgEnv-cray is missing key ARM intrinsics that prevent Chapel from compiling.